[2024-09-01T10:49:07.821+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-09-01T10:49:08.261+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: simple_dag.send_failure_email manual__2024-09-01T10:48:58.073286+00:00 [queued]>
[2024-09-01T10:49:08.299+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: simple_dag.send_failure_email manual__2024-09-01T10:48:58.073286+00:00 [queued]>
[2024-09-01T10:49:08.305+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 2
[2024-09-01T10:49:08.350+0000] {taskinstance.py:2330} INFO - Executing <Task(EmailOperator): send_failure_email> on 2024-09-01 10:48:58.073286+00:00
[2024-09-01T10:49:08.372+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:62: DeprecationWarning: This process (pid=6376) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-09-01T10:49:08.377+0000] {standard_task_runner.py:64} INFO - Started process 6378 to run task
[2024-09-01T10:49:08.380+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'simple_dag', 'send_failure_email', 'manual__2024-09-01T10:48:58.073286+00:00', '--job-id', '382', '--raw', '--subdir', 'DAGS_FOLDER/main.py', '--cfg-path', '/tmp/tmp_kmku4ai']
[2024-09-01T10:49:08.386+0000] {standard_task_runner.py:91} INFO - Job 382: Subtask send_failure_email
[2024-09-01T10:49:08.506+0000] {task_command.py:426} INFO - Running <TaskInstance: simple_dag.send_failure_email manual__2024-09-01T10:48:58.073286+00:00 [running]> on host 19bb8daff211
[2024-09-01T10:49:08.787+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_EMAIL='jorgepons90@gmail.com,facturasjorgepons@gmail.com' AIRFLOW_CTX_DAG_OWNER='Jorge Pons' AIRFLOW_CTX_DAG_ID='simple_dag' AIRFLOW_CTX_TASK_ID='send_failure_email' AIRFLOW_CTX_EXECUTION_DATE='2024-09-01T10:48:58.073286+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-09-01T10:48:58.073286+00:00'
[2024-09-01T10:49:08.790+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-09-01T10:49:08.881+0000] {base.py:84} INFO - Using connection ID 'smtp_default' for task execution.
[2024-09-01T10:49:08.883+0000] {email.py:271} INFO - Email alerting: attempt 1
[2024-09-01T10:49:09.451+0000] {email.py:282} INFO - Sent an alert email to ['algenet4.jpons@gmail.com']
[2024-09-01T10:49:10.420+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-09-01T10:49:10.438+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=simple_dag, task_id=send_failure_email, run_id=manual__2024-09-01T10:48:58.073286+00:00, execution_date=20240901T104858, start_date=20240901T104908, end_date=20240901T104910
[2024-09-01T10:49:10.490+0000] {local_task_job_runner.py:243} INFO - Task exited with return code 0
[2024-09-01T10:49:10.520+0000] {taskinstance.py:3503} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-09-01T10:49:10.524+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
